### PR TITLE
Icon props (next / previous)

### PR DESCRIPTION
**Pull request by service-paradis**
## Proposed Changes#

- Add default config for next/previous icon (used in Steps, Datepicker and Pagination)
- [Steps] Use default config for next/previous button icons. Also add a prop into the component
- [Datepicker] Use default config for next/previous button icons. Also add a prop into the component
- [Pagination] Use default config for next/previous button icons. Also add a prop into the component

### CODE REVIEW

File: src/components/pagination/Pagination.vue [hunk 1]
```diff
@@ -134,6 +137,13 @@ export default {
             return parseInt(this.rangeAfter)
         },
 
+        newIconPrev() {
+            return this.iconPrev || config.defaultIconPrev
```
Comment (jtommy) [inline src/components/pagination/Pagination.vue]: Why do we need this ? I think you might add  config as default value of the prop (so in other components)

Comment (service-paradis) [reply src/components/pagination/Pagination.vue]: Sure!
It was done this way in some other components.

Fixed!

**Comment by jtommy** [comment]
It's ok but I don't want to add an example for each components, for example you might remove datepicker example

**Comment by service-paradis** [comment]
Removed!
https://github.com/buefy/buefy/pull/1615/commits/a7cf76cde0a21423df10ba81fde1140b04c43d08

**Comment by jtommy** [comment]
Please check conflicts

**Comment by service-paradis** [comment]
Sorry for the conflict. CircleCI take a long time to run before I can see if there are any conflicts. Should be resolved now, but waiting to see CircleCI results.

**Comment by service-paradis** [comment]
@jtommy Fixed!

**Pull request merged**